paper-api/src/main/java/org/bukkit/World.java

```diff
@@ -4168,6 +4168,30 @@ default Location locateNearestBiome(@NotNull Location origin, @NotNull Biome bio
     void sendGameEvent(@Nullable Entity sourceEntity, @NotNull GameEvent gameEvent, @NotNull Vector position);
     // Paper end
 
+    // Paper start - expose potion mixing API
```

Comment:
We are hardforked. No need for comments anymore.

Reply:
Ah, that is only for diff in mojang classes :sweat_smile: when writing patches. you are just editing normal .java files, maybe we need to clarify that.